### PR TITLE
chore(project): move dependencies to devDependencies

### PR DESCRIPTION
**Pull request by joshblack**
Addresses part of #177 

Move `archiver` and `git-branch` to `devDependencies`.

### CODE REVIEW

**Comment by vpicone** [comment]
I'm not sure if this would technically be a "breaking change."

**Comment by joshblack** [comment]
@vpicone I think as long as we keep the web-based font files we should be good, and provide a link to download sources 👍

File: package.json [hunk 1]
```diff
@@ -19,12 +19,10 @@
...
     "prepare": "yarn build"
   },
-  "dependencies": {
-    "archiver": "^3.0.0",
-    "git-branch": "^2.0.1"
-  },
   "devDependencies": {
+    "archiver": "^3.0.0",
     "fs-extra": "^7.0.0",
+    "git-branch": "^2.0.1"
```
Comment (akx) [inline package.json]: Syntax error here - gotta have a comma at the end of this line.

Comment (joshblack) [reply package.json]: Thanks @akx! That's what I get for editing through GitHub UI 😂

**Pull request merged**